ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCUI-nonprod.json

```diff
@@ -299,9 +297,29 @@
     "AccessControl": [
       {
         "UserRoles": [
-          "caseworker-civil-systemupdate",
+          "cui-admin-profile"
+        ],
+        "CRUD": "CRU"
+      },
+      {
+        "UserRoles": [
+          "APP-SOL-SPEC-PROFILE",
```

Comment:
wouldn't you add the defendant role there to be able to read?

Reply:
done